src/assets/mock-categories-cards.json

```diff
@@ -1,32 +1,209 @@
 [
   {
     "id": 1,
-    "title": "Спорт",
-    "picture": "assets/icons/sport.png"
+    "title": "Спорт"
   },
   {
     "id": 2,
-    "title": "Мистецтво",
-    "picture": "assets/icons/art.png"
+    "title": "Мистецтво"
   },
   {
     "id": 3,
-    "title": "Конструювання",
-    "picture": "assets/icons/robo.png"
+    "title": "Конструювання"
   },
   {
     "id": 4,
-    "title": "Драматургія",
-    "picture": "assets/icons/drama.png"
+    "title": "Драматургія"
   },
   {
     "id": 5,
     "title": "Музика",
-    "picture": "assets/icons/music.png"
+    "subcategories": [
```

Comment:
needs to discuss this, since it was requested to have possibility to edit categories so to keep them on FE side is not good idea